change/@azure-msal-node-2020-10-26-20-16-45-remove-debug.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Since you are changing behavior inside the library
```suggestion
  "type": "prerelease",
```